genestack_client/genestack_connection.py

```diff
@@ -253,16 +255,14 @@ def get_response(self, method, params=(), trace=True):
         :return: Response object
         :rtype: Response
         """
-        post_data = {'method': method}
-        if trace:
-            post_data['trace'] = True
+        post_data = {}
         if params:
             post_data['parameters'] = json.dumps(params)
 
-        path = '/application/invoke/%s' % self.application_id
+        path = '/application/invoke/%s/%s' % (self.application_id, urllib.quote(method))
```

Comment:
Should we check that method is not unicode, and otherwise convert it with `encode('utf-8')`?